app-backend/api/src/main/scala/thumbnail/Routes.scala

```diff
@@ -56,6 +42,7 @@ trait ThumbnailRoutes extends Authentication
     }
   }
 
+  @SuppressWarnings(Array("AsInstanceOf"))
```

Comment:
Seems like there should be a straightforward way to actually fix this vs. suppress the warning, but I was unable to sort out how to make a non-binary `MediaType` work with `ContentType`.